ResourcesCard.jsx

```diff
@@ -1 +1,83 @@
+import React, { useEffect } from "react";
+
+import { sendEvt } from "services/event.service";
+
+import imgFallback from "img/resource_fallback.png";
+
+import styles from "./ResourcesCard.module.scss";
+
+const ResourcesCard = () => {
+  const [resources, setResources] = useState([]);
+  const [primaryResource, setPrimaryResource] = useState(none);
+  const [secondaryResources, setSecondaryResources] = useState([]);
+
+  var sendEventTracking = (resourceUuid) => {
+    const evt = new Object();
+    evt = {
+      event_name: "web_dashboard_card_" + r.id + "_" + r.title + "_tap"
+    };
+    sendEvt(evt);
+    console.log("sending event");
+  };
+
+  // TODO
+  useEffect(async () => {
+    let req = await getResources(userId);
+    setResources(req.resources);
+
+    let primaryResource = resources[0];
+    let secondaryResources = resources.slice(1, 5);
```

Comment:
I would recommend using `const` here instead of let, since we're not reassigning.

Also, I wonder if it would be a little less confusing if called this `primaryResource` and `secondaryResources` something different so that we don't confuse them with the state. Or, potentially, we can eliminate these variables and just do 
```
setPrimaryResource(resources[0]);
   setSecondaryResources(resources.slice(1, 5));
```